tracer/test/test-applications/integrations/Samples.RabbitMQ/Program.cs

```diff
@@ -26,118 +26,93 @@ private static string Host()
 
         public static void Main(string[] args)
         {
-            RunRabbitMQ();
+            // Test a derived type for the sync consumer from the library
```

Comment:
Should we also update our RabbitMQ sample (the one in the doc) to advertize this use case? Or at least add a comment stating its possible? I assume we should update the public doc as well, could you prepare a PR on the doc repo and add the `needs-doc-update` label to this PR?

Reply:
Yes updating the RabbitMQ sample in this repo is a good idea, so I'll do that.

I'm not sure if we need to update the public docs with anything. Basically, customers looking at our integration for the `RabbitMQ.Client` NuGet package to work without any edge cases, and this feature is more-or-less a bug fix, so now it works as expected

Reply:
We can maybe add a comment in the public doc stating in what version we have fixed it. We had 2 escalations on RabbitMQ, so I assume we shouldn't be shy with more info. But I let you judge what's best, I'm always the one being corrected for docs, so I'm not the best judge I believe  ;)

Reply:
I'm going to follow-up with a PR that shows we support multiple versions of EasyNetQ, which I think should clear up any ambiguity 😄